packages/env/lib/build-docker-compose-config.js

```diff
@@ -113,6 +113,34 @@ module.exports = function buildDockerComposeConfig( config ) {
 	const developmentPorts = `\${WP_ENV_PORT:-${ config.env.development.port }}:80`;
 	const testsPorts = `\${WP_ENV_TESTS_PORT:-${ config.env.tests.port }}:80`;
 
+	// Set the WordPress, WP-CLI, PHPUnit PHP version if defined.
+	const phpVersion = config.env.development.phpVersion
+		? config.env.development.phpVersion.toString()
```

Comment:
I think we should make sure it is a string at the parse/load step, and then we can rely on it being a string here :)

We could also default `phpVersion` to be an empty string so that we don't have to check for its existence here

Reply:
I started the change and performed the `toString()` on parse/load but I'm resistant to have the default be an empty string because I feel like an empty string is technically harder to assume that was never set to begin with versus it maybe was intentionally set to an empty string.

Reply:
Actually, attempting to perform a toString() on parse requires explicit checking also for whether it's set. Since the configuration validation is more complete and ensures the value, when set, is a string there is no need for this later.